### ISSUE TITLE
Crash when add a new predefined filter

### ISSUE THREAD

**Issue by onionhead0708** [variar/klogg#342]
Details for the issue
--------------------

#### What did you do?
In the Tools -> Predefined Filters, add a new one with following information and click on the OK / Apply button.
Name = BaseStartUp
Patter = BaseStartUp - Common LogFactory initialized successfully

-------------------------
Crash id:
b2b0eed9-d306-47c7-a9de-010dc985c3e3
-------------------------
Useful extra information
-------------------------
> Klogg version 21.06.0.1026 (built on 2021-06-09 from commit 4c29f49) [built for x86_64-little_endian-llp64]
> running on Windows 10 Version 1909 (winnt/10.0.18363) [x86_64]
> and Qt 5.15.2

**Comment by variar** [comment]
Thanks for reporting this issue. I need to add more automated tests. 

```
Crashed Thread: 7332

Application Specific Information:
Fatal Error: EXCEPTION_ACCESS_VIOLATION_READ

Thread 7332 Crashed:
0   Qt5Widgets.dll                  0x7fff9978f3d0      QAbstractButton::isChecked
1   klogg.exe                       0x7ff6338d978b      PredefinedFiltersDialog::readFiltersTable (predefinedfiltersdialog.cpp:140)
2   klogg.exe                       0x7ff6338d967d      PredefinedFiltersDialog::saveSettings (predefinedfiltersdialog.cpp:120)
3   klogg.exe                       0x7ff6338da613      PredefinedFiltersDialog::resolveStandardButton (predefinedfiltersdialog.cpp:247)
```

**Comment by variar** [comment]
@onionhead0708 this should be fixed in 21.06.0.1030
